metashare/api/jobs.py

```diff
@@ -128,6 +131,34 @@ def alert_user_about_expiring_org(*, org, days):
         )
 
 
+def _get_valid_target_directories(scratch_org_config):
+    """
+    Expects to be called from within a `local_github_checkout`.
+    """
+    if scratch_org_config.get("source_format") == "sfdx":
+        with open("sfdx-project.json") as f:
+            sfdx_project = json.loads(f.readlines())
+            package_directories = sfdx_project["packageDirectories"]
```

Comment:
@wlonk It looks like you're expecting this to be a list of strings, but according to the docs David linked (https://developer.salesforce.com/docs/atlas.en-us.sfdx_dev.meta/sfdx_dev/sfdx_dev_ws_config.htm) it seems like it's actually a list of objects. We also might want to track which one is the default?